qa/L0_perf_analyzer/test.sh

```diff
@@ -1094,6 +1094,34 @@ set -e
 kill $SERVER_PID
 wait $SERVER_PID
 
+
+# Test the HTTP protocol on a unix socket
+UNIX_PORT="unix:///run/domain.unix"
+SERVER_ARGS="--model-repository=${DATADIR} --http-address=${UNIX_PORT}"
+
+run_server
+if [ "$SERVER_PID" == "0" ]; then
+    echo -e "\n***\n*** Failed to start $SERVER\n***"
+    cat $SERVER_LOG
+    exit 1
+fi
+
+set +e
+
+$PERF_ANALYZER -v -u ${UNIX_PORT} -i http -m graphdef_int32_int32_int32 \
+    -s ${STABILITY_THRESHOLD} \
+    > ${CLIENT_LOG}.https_success 2>&1
+if [ $? -ne 0 ]; then
+    cat ${CLIENT_LOG}.https_success
+    RET=1
+fi
+
+set -e
+
+kill $SERVER_PID
+wait $SERVER_PID
+
+
```

Comment:
Can you also manually clean up the UNIX port? Since the unix socket will not be destructed after the test is complete.

`rm /run/domain.unix`

Reply:
updated